Zombie/src/main/java/dk/gruppeseks/bodtrd/zombie/ZombiePlugin.java

```diff
@@ -41,29 +41,27 @@
     @Override
     public void start(World world)
     {
+
         Installer.Plugin = this;
 
         ViewManager.createView(ZOMBIE_IMAGE_FILE_PATH, false);
 
         _world = world;
         _zombies = new HashMap();
 
-        for (int i = 0; i < 10; i++)
+        for (int i = 0; i < 2; i++)
         {
             Entity zombie = createZombieEntity();
             _zombies.put(zombie.getID(), zombie);
             world.addEntity(zombie);
         }
 
-        _processor = new ZombieProcessor(_zombies);
+        _ai = _lookup.lookup(AISPI.class);
```

Comment:
do we plan on being able to swap this component in runtime? its the same issue as with MapSPI atm. we can load an implementation but we cannot swap it or unload it